docs/docs/api/06-Configuration/askui-ui-controller.md

```diff
@@ -8,14 +8,19 @@ custom_edit_url: null
 
 ### Properties
 
-- [binaryVersion](#binaryversion)
-- [display](#display)
-- [host](#host)
-- [logFilePath](#logfilepath)
-- [logLevel](#loglevel)
-- [minimize](#minimize)
-- [overWriteBinary](#overwritebinary)
-- [port](#port)
+ [askui UI Controller](#askui-ui-controller)
+  - [Table of Contents](#table-of-contents)
+    - [Properties](#properties)
```

Comment:
Like already commented, I don't like it as it seems redundant but maybe you have good reason. Can you explain it to me?

Reply:
Actually I looked at @mlikasam-askui PR 1034 and his changes to avoid conflicts. Saw his change and added it . Also used the same schema for the client.  Think it was fine as overview on the first look but the value of the three links are probably not so high